### PR TITLE
Search

### PR DESCRIPTION
**Pull request by lxc150896**
fix search

### CODE REVIEW

File: app/Repositories/Eloquents/BookEloquentRepository.php [hunk 1]
```diff
@@ -20,6 +20,7 @@ class BookEloquentRepository extends AbstractEloquentRepository implements BookR
         'description',
         'avg_star',
         'slug',
+        'author'
```
Comment (framgiaciv3) [inline app/Repositories/Eloquents/BookEloquentRepository.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoCommaAfterLast</br>**Severity**: error</br>**File**: app/Repositories/Eloquents/BookEloquentRepository.php</br>**Line**: 23</br>Comma required after last value in array declaration

Comment (framgiaciv3) [inline app/Repositories/Eloquents/BookEloquentRepository.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoCommaAfterLast</br>**Severity**: error</br>**File**: app/Repositories/Eloquents/BookEloquentRepository.php</br>**Line**: 23</br>Comma required after last value in array declaration

File: app/Http/Controllers/HomeController.php [hunk 2]
```diff
@@ -79,33 +80,25 @@ public function changeLanguage(Request $request, $language)
...
 
-    public function searchAjax(Request $request)
+    public function searchAjax(SearchRequest $request)
     {
-        $users = null;
-        if (Auth::check()) {
-            $users = $this->user->search('name', $request->req);
+        if ($request->req != '') {
+            $result = collect([
+                'description' => $this->book->search('description', $request->req),
```
Comment (danhloimta) [inline app/Http/Controllers/HomeController.php]: search theo description à?

File: resources/assets/user/css/custom.css [hunk 3]
```diff
@@ -935,3 +935,37 @@ ul.suggestions-list.noti-list .view-all {
     text-align: right;
     padding: 6px 22px 6px 0;
 }
+.suggestion ul li:hover {background: #ededed;}
```
Comment (danhloimta) [inline resources/assets/user/css/custom.css]: convention

File: resources/assets/user/css/custom.css [hunk 4]
```diff
@@ -935,3 +935,37 @@ ul.suggestions-list.noti-list .view-all {
...
+.line {
+    border-top: 1px solid #CCCCCC;
```
Comment (danhloimta) [inline resources/assets/user/css/custom.css]: viết chữ thường

File: resources/assets/user/css/custom.css [hunk 5]
```diff
@@ -935,3 +935,37 @@ ul.suggestions-list.noti-list .view-all {
...
+    width: 50px;
+    border-radius: 3%;
+}
+.suggestion-search {
+    border: 1px solid #DDDDDD;
+    font-weight: 400;
+}
+.suggestion-search ul li {
+}
+.suggestion-search ul li:hover {background: #ededed;}
```
Comment (danhloimta) [inline resources/assets/user/css/custom.css]: tương tự

File: resources/views/search.blade.php [hunk 6]
```diff
@@ -21,7 +32,7 @@
             <div class="row">
                 <div class="col-lg-12">
                     <div class="entry-header-title">
-                        <h2>{{ trans('settings.header.results') }}</h2>
+                        <h2>{{ trans('page.keySearch')}} {{ $key }}</h2>
```
Comment (danhloimta) [inline resources/views/search.blade.php]: convention

File: resources/assets/user/css/custom.css (hunk 5 above)
Comment (BunnyPi04) [inline resources/assets/user/css/custom.css]: convention

File: resources/assets/user/css/custom.css [hunk 7]
```diff
@@ -935,3 +935,37 @@ ul.suggestions-list.noti-list .view-all {
...
+    max-width: inherit;
+    width: 50px;
+    border-radius: 3%;
+}
+.suggestion-search {
+    border: 1px solid #DDDDDD;
+    font-weight: 400;
+}
+.suggestion-search ul li {
+}
```
Comment (BunnyPi04) [inline resources/assets/user/css/custom.css]: ko có gì thì sao ghi vào làm gì?

File: resources/assets/user/css/style.css [hunk 8]
```diff
@@ -6589,3 +6588,5 @@ p.lost-password a:hover {
     padding-top: 10px;
     float: right;
 }
+.suggestion ul li:hover {background: #DDDDDD;}
+.suggestion ul li:hover ul {display: block;}
```
Comment (BunnyPi04) [inline resources/assets/user/css/style.css]: viết thành khối

File: resources/views/search.blade.php [hunk 9]
```diff
@@ -5,14 +5,25 @@
...
                         <ul>
                             <li><a href="{{ route('home') }}">{{ trans('settings.default.home') }}</a></li>
                             <li><a class="active">{{ trans('settings.header.search') }}</a></li>
                         </ul>
                     </div>
                 </div>
+                <div class="col-lg-8">
+                    <div class="breadcrumbs-menu">
+                        <ul>
+                            <li>{{ trans('page.searchBy')}}</li>
```
Comment (BunnyPi04) [inline resources/views/search.blade.php]: convention

**Pull request merged**